src/sparc/client/services/_default.py

```diff
@@ -0,0 +1,28 @@
+import os
+from abc import ABC, abstractmethod
+
+
+class ServiceBase(ABC):
+    @abstractmethod
+    def __init__(self, connect, *args, **kwargs) -> None:
```

Comment:
shouldn't ``config`` be also an explicit key argument in the constructor?

Reply:
> Looks good to me. i will submit the osparc interface implementation asap
> 
>     * I would suggest adding a pre-commit config to get all the work the reviewdoc done before we push. I can create a pull request if you like

That would be great!

Reply:
> shouldn't `config` be also an explicit key argument in the constructor?

Good point! We can add it - it may be useful for the modules to capture it.